content/_changelogs/10.0.0.md

```diff
@@ -0,0 +1,331 @@
+  - A warning message is now displayed when `cypress open` is ran with the
+    `--browser` CLI flag for a browser that does not exist on the system.
+    Addressed in [#18473](https://github.com/cypress-io/cypress/pull/18473).
+  - If a testing type and browser flag are passed to `cypress open` when
+    launching a specific project, the browser will automatically launch.
+    Addressed in [#21538](https://github.com/cypress-io/cypress/pull/21538).
+
+- The `cypress open` experience has been updated to take you through our
+  **Launchpad** with many features detailed below:
+
+  - When you launch Cypress via `cypress open` you'll be asked whether you want
+    to do end-to-end or component testing and taken through the configuration
+    necessary to setup your project. Addressed in
+    [#18924](https://github.com/cypress-io/cypress/pull/18924).
+  - Having files scaffolded on new projects during end-to-end testing is now opt
+    in and not automatic. Addressed in
+    [#18924](https://github.com/cypress-io/cypress/pull/18924).
+
+  - When Cypress detects you have a `cypress.json` during `cypress run`, it will
+    error and ask you to run `cypress open` to migrate. When Cypress detects you
+    have a `cypress.json` during `cypress open`, it will guide you through
+    project migration to ensure your project continues to work as expected.
+    Addressed in [#18924](https://github.com/cypress-io/cypress/pull/18924).
+
+- After launching the browser, there are several new feature in our **App**
+  detailed below:
+
+  - The Cypress App has been redesigned so that the Command Log and all
+    associated navigation displays with a dark background.
+  - You can now change specs or the browser within the same window as running
+    your tests.
+  - Many git details are now shown within your current project in Cypress
+    including the current branch and the last commit details of each spec in the
+    spec explorer. Addressed in
+    [#20648](https://github.com/cypress-io/cypress/pull/20648),
+    [#21139](https://github.com/cypress-io/cypress/pull/21139),
+    [#21436](https://github.com/cypress-io/cypress/pull/21436) and
+    [#20744](https://github.com/cypress-io/cypress/pull/20744)
+  - When creating a new spec within Cypress, your project's language is detected
+    to provide the correct file extension and the `specPattern` is checked to
+    ensure the spec is found by Cypress. After creating the spec, you can
+    preview the spec that was generated with the option to immediately run the
+    spec or create another spec. Addressed in
+    [#21225](https://github.com/cypress-io/cypress/pull/21225),
+    [#20567](https://github.com/cypress-io/cypress/pull/20567),
+    [#20657](https://github.com/cypress-io/cypress/pull/20657),
+    [#20962](https://github.com/cypress-io/cypress/pull/20962).
+  - The Settings page has been redesigned to more clearly distinguish between
```

Comment:
Should we call out that the Settings page and Runs page are both now found in the main app, and no longer found in the Launchpad?